examples/demo.js

```diff
@@ -28,6 +32,33 @@ window.onload = function () {
     }
   };
 
+  function displayVersionNumber() {
+    $('#version-number-text').text("Image Sequencer v" + currentVersionNumber);
+  }
+  displayVersionNumber();
+
+  // Get the current version number from package.json on GitHub.
+  function getLatestVersionNumber() {
+    var request = new XMLHttpRequest();
+    request.onreadystatechange = function() {
+      if(request.readyState == 4 && request.status == 200) {
+        var response = JSON.parse(this.responseText);
+        latestVersionNumber = JSON.parse(atob(response.content)).version;
+        console.log("The latest NPM version number from GitHub is: " + latestVersionNumber);
+        if(true /* Local version is less than latest NPM version */) { //TODO: change to semver.lt(currentVersionNumber, latestVersionNumber) once semver is successfully included
+          console.log("Image sequencer is not on the latest version and should be updated.");
+        } else {
+          console.log("Image sequencer is up to date with the latest version.");
+        }
+      }
+    }
+    request.open("GET", "https://api.github.com/repos/publiclab/image-sequencer/contents/package.json", true);
```

Comment:
This is cool. So you want to clear the cache(via demo.js) if a new version is available?

Reply:
Yeah, but I'll probably end up doing this through the service worker.

Reply:
This is not a bad way either. Is there a downside to this?